roles/servicetelemetry/templates/manifest_prometheus.j2

```diff
@@ -48,8 +48,8 @@ spec:
     - -upstream=http://localhost:9090/
     - -cookie-secret-file=/etc/proxy/secrets/session_secret
     - -openshift-service-account=prometheus-stf
-    - '-openshift-sar={"namespace":"{{ ansible_operator_meta.namespace }}","resource": "prometheus", "group":"{{ prometheus_operator_api_string | replace("/v1","") }}", "verb":"get"}'
-    - '-openshift-delegate-urls={"/":{"namespace":"{{ ansible_operator_meta.namespace }}","resource": "prometheus", "group":"{{ prometheus_operator_api_string | replace("/v1","") }}", "verb":"get"}}'
+    - '-openshift-sar={"namespace":"{{ ansible_operator_meta.namespace }}","resource": "prometheuses", "resourceAPIGroup":"{{ prometheus_operator_api_string | replace("/v1","") }}", "verb":"get"}'
+    - '-openshift-delegate-urls={"/":{"namespace":"{{ ansible_operator_meta.namespace }}","resource": "prometheuses", "group":"{{ prometheus_operator_api_string | replace("/v1","") }}", "verb":"get"}}'
```

Comment:
This change and the change to the prometheus-reader Role above are just to keep things consistent between the (SAR-based) interactive logins, and (token-based) delegated logins, and to prevent surprises if you're using the `oc create role` command line tool which force everything to plural. We can use "prometheus" or "prometheuses" in these definitions, as long as they match.